apps/elasticsearch/deployers/test_vars.yml

```diff
@@ -0,0 +1,11 @@
+application_deployment: es-statefulset.yaml
+application_rbac: esearch_rbac.yml
+app_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+app_label: "{{ lookup('env','APP_LABEL') }}"
+test_name: elasticsearch-deployment
+app_replica: "{{ lookup('env','APP_REPLICA') }}"
+application_name: "Elasticsearch"
+action: "{{ lookup('env','ACTION') }}"
+app_pvc: "{{ lookup('env','APP_PVC') }}"
+storage_class: "{{ lookup('env','PROVIDER_STORAGE_CLASS') }}"
+operator_ns: openebs
```

Comment:
Where do we use operator_ns?

Reply:
@gprasath we are using it while deletion / de-provisioning of application